check/request.go

```diff
@@ -0,0 +1,10 @@
+package check
+
+type VersionCheckRequest struct {
+	Product string
+	Version string
+	Arch    string
+	OS      string
+	DB      string
+	Token   string
```

Comment:
```suggestion
	Token   []byte
```
?

Reply:
It gets passed as a get URL parameter, I would prefer to keep it a string.

Reply:
ok